open_academy/models/open_academy_course.py

```diff
@@ -9,3 +9,16 @@ class OpenAcademyCourse(models.Model):
     description = fields.Text()
     responsible_id = fields.Many2one("res.users")
     sessions_ids = fields.One2many("open.academy.session", "course_id")
+
+    _sql_constraints = [
+        ("title_description_diferents", "check(title!=description)",
+        "the title and the description must be diferents"),("title_unique", "unique(title)",
+        "this course title is already taken "),
+    ]
```

Comment:
Please use uppercase for SQL sentences. The message must be legible for the user, please use good spelling for this.
```suggestion
    _sql_constraints = [
        ("title_description_different", "CHECK(title!=description)",
        "The title and description must be different."),
        ("title_unique", "UNIQUE(title)",
        "This course title is already taken."),
    ]
```